### PR TITLE
Updated some logos and favicons

### PR DESCRIPTION
**Pull request by evrifaessa**
Please carefully review. It's midnight and I might've missed something. Thanks.

### CODE REVIEW

**Comment by John-S4** [comment]
Why are you using such strange background sizing instead of whole pixel values?

`background-size: 117.12px 138.24px;`

**Comment by jaapmarcus** [comment]
Also prefer if .svg is used instead of .png as it scales better

**Comment by John-S4** [comment]
> Also prefer if .svg is used instead of .png as it scales better

Agreed

File: src/bb-load.php [hunk 1]
```diff
@@ -129,8 +129,9 @@ function handler_exception($e)
               display: inline-block;
               width: 140px;
               height: 140px;
-              background-image: url('/bb-themes/boxbilling/assets/images/box.png');
-              background-size: cover;
+              background-image: url('/bb-themes/boxbilling/assets/images/fb_wallet.png');
```
Comment (jaapmarcus) [inline src/bb-load.php:132]: Why .png and not .svg?

File: src/bb-load.php [hunk 2]
```diff
@@ -129,8 +129,9 @@ function handler_exception($e)
...
+              background-repeat: no-repeat;
+              background-size: 117.12px 138.24px;
```
Comment (jaapmarcus) [inline src/bb-load.php:134]: Why not rounded numbers? Doesn't it create any strange artifacts?

**Review by jaapmarcus — CHANGES_REQUESTED** [review]
It also makes sense to have a "dark mode" version available in the repo when needed.

**Comment by evrifaessa** [comment]
> Why are you using such strange background sizing instead of whole pixel values?
> 
> `background-size: 117.12px 138.24px;`

It's the 40% of the original size. I'll get the circle bigger and make the logo 50% to make the numbers integers.

**Comment by evrifaessa** [comment]
> It also makes sense to have a "dark mode" version available in the repo when needed.

Two versions of the wordmark exist both in the branding repo and the theme assets.

image

**Comment by evrifaessa** [comment]
Also uploaded the svg versions and used them wherever I can.

**Comment by evrifaessa** [comment]
One thing left is this grayscale BoxBilling logo in the footer of the boxbilling theme.
I don't know what to replace this with.

image

**Comment by evrifaessa** [comment]
Another thing is this icon that comes from the branding module

image

**Review by jaapmarcus — APPROVED** [review]

**Comment by John-S4** [comment]
> One thing left is this grayscale BoxBilling logo in the footer of the boxbilling theme. I don't know what to replace this with.
> 
> image

I'd be tempted to use the logo with black text, and apply a CSS greyscale filter.

**Comment by chippytech** [comment]
What about this
download (3)

**Comment by John-S4** [comment]
Just use one of the existing files and apply a grayscale filter if it needs to be without colour, otheriwse just use the logo with colour in the footer, then let's get it merged and rid of that old bb logo popping up everywhere.

**Review by John-S4 — APPROVED** [review]
All looks good, just do something with that footer logo and then merge it.

**Comment by evrifaessa** [comment]
With a full grayscale effect and 50% opacity, the footer logo looks like this:

image

**Pull request merged**